<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>711</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Edition Des Chèques</string>
    </attribute>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>300</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Un email sera envoyé au client </string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Un email sera envoyé au client </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mod__endosse">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>280</y>
       <width>70</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Endossé ?</string>
     </property>
    </widget>
    <widget class="QUndoView" name="mod_num_cheque">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>256</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Banque</string>
     </property>
    </widget>
    <widget class="QLabel" name="Logo1_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>-60</y>
       <width>471</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>170</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Client</string>
     </property>
    </widget>
    <widget class="QUndoView" name="mod_client">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>200</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro du chèque</string>
     </property>
    </widget>
    <widget class="QUndoView" name="mod_banque">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>256</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Montant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mod_montant">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>250</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date d’èchéance</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="mod_date">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <zorder>Logo1_2</zorder>
    <zorder>commandLinkButton</zorder>
    <zorder>label</zorder>
    <zorder>mod__endosse</zorder>
    <zorder>mod_num_cheque</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>mod_client</zorder>
    <zorder>label_4</zorder>
    <zorder>mod_banque</zorder>
    <zorder>label_5</zorder>
    <zorder>mod_montant</zorder>
    <zorder>label_6</zorder>
    <zorder>mod_date</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
